src/CompoundV3YieldOracle.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: BUSL 1.1
+pragma solidity 0.8.13;
+
+import "./interfaces/ICompoundV3YieldOracleAdmin.sol";
+import "./interfaces/IComet.sol";
+// import "./utils/Admin.sol";
+
+contract CompoundV3YieldOracle is ICompoundV3YieldOracleAdmin /*, Admin */ {
+    /**
+     * /////////// CONSTANTS ///////////////
+     */
+    address public constant COMET_USDC_ADDRESS = 0xc3d688B66703497DAA19211EEdff47f25384cdc3;
```

Comment:
Why USDC instead of DAI? I prefer DAI from a stability/decentralization perspective.

Reply:
see response to other comment. didn't want to go with compound v2 just to get DAI. let me check the comet roadmap and see "DAI when" and I can re-gear this with Aave/DAI if you feel it necessary.